tracer/test/snapshots/ProcessStartTests.SubmitsTraces.verified.txt

```diff
@@ -0,0 +1,122 @@
+﻿[
+  {
+    TraceId: Id_1,
+    SpanId: Id_2,
+    Name: command_execution,
+    Resource: nonexisting1.exe,
+    Service: Samples.ProcessStart-command,
+    Type: system,
+    Error: 1,
+    Tags: {
```

Comment:
Can you update the test so at least one of the created spans has the new tag `cmd.environment_variables`?

Reply:
Also, can you produce a span that does not have an error? Maybe you can do a runtime check in the sample application for the operating system and then do a different application based on the result?

Reply:
I have added the environment variables to the test. About launching real processes, I do not feel confortable doing that in CI environments. I think that is cleaner if the test doesn't launch any external process in the hosting machine. What do you think?

Reply:
Yeah that makes sense, I support that